components/sections/CompanyOverview/index.jsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect, useState } from "react";
+import ArrowIcon from "../../ui/ArrowIcon";
+import Overview from "./Overview";
+import { motion as Motion } from "framer-motion";
+import companyTabs from "./../../../data/companyTabs";
+import Wrapper from "../../Wrapper";
+
+const CompanyOverview = () => {
+  const [activeTab, setActiveTab] = useState(companyTabs[0]);
+  const [activeTabIndex, setActiveTabIndex] = useState(0);
+  const changeActiveTabMobile = (tab) =>
+    tab === activeTabIndex ? setActiveTabIndex(null) : setActiveTabIndex(tab);
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerWidth >= 640 && !activeTabIndex) {
+        setActiveTab(companyTabs[0]);
+        setActiveTabIndex(0);
+      }
+    };
+    window.addEventListener("resize", handleResize);
+    return () => window.removeEventListener("resize", handleResize);
+  }, [activeTab, activeTabIndex]);
+
+  return (
+    <Wrapper>
+      <div className="justify-end hidden mb-17.7 sm:mt-33 lg:mt-35.55 xl:mt-24 sm:flex">
+        <div className="sm:w-full lg:w-757 xl:w-811">
+          <ul className="flex gap-10.5 items-center py-5 relative">
+            {companyTabs.map((item, index) => (
+              <li className="relative space-y-5" key={index}>
+                <button
+                  className={`cursor-pointer text-2xl leading-extra -tracking-tighter transition-all ease-in-out duration-300 ${
+                    index === activeTabIndex
+                      ? "text-purple "
```

Comment:
Avoid unnecessary spaces in the class naming
```suggestion
                      ? "text-purple"
```